lineapy/external.annotations.yaml

```diff
@@ -74,29 +74,61 @@
 - module: pandas
   annotations:
     - criteria:
-        class_method_name: to_sql
+        class_method_names: 
+          - to_sql
+          - to_gbq
         class_instance: DataFrame
```

Comment:
can you see if we can go to the base class here so that we capture both series and dataframe (it might be NDFrame as well.) there is another annotation in here that uses pandas.core.generic module you can use as reference.